pkg/aws/validations_internal.go

```diff
@@ -131,3 +133,50 @@ func (ac *awsCloud) validateRemoveTag(subnetID *string) error {
 
 	return determinePermissionError(err, "delete tags from subnets")
 }
+
```

Comment:
This file name should remain _validations.go_.  It's only the test file that should contain "__internal_" to satisfy the linter.

Reply:
sorry, the IDE tricked me :)